ee/ucp/admin/configure/restrict-services-to-worker-nodes.md

```diff
@@ -1,40 +1,37 @@
 ---
 title: Restrict services to worker nodes
-description: Learn how to configure Universal Control Plane to only allow running services in worker nodes.
+description: Learn how to configure Docker Universal Control Plane to only allow running services in worker nodes.
 keywords: ucp, configuration, worker
 ---
 
 >{% include enterprise_label_shortform.md %}
 
-You can configure UCP to allow users to deploy and run services only in
-worker nodes. This ensures all cluster management functionality stays
-performant, and makes the cluster more secure.
+Docker Universal Control Plane (UCP) is set to run on both manager and worker nodes by default, however, it can be configured to run only on worker nodes. This ensures that all cluster management functionality stays performant, and makes the cluster more secure.
```

Comment:
I suggest editing the last sentence to read: "This ensures that all cluster management functionality stays performant, while also serving to make the cluster more secure.